airflow/models/connection.py

```diff
@@ -145,6 +145,38 @@ def parse_from_uri(self, uri):
         if uri_parts.query:
             self.extra = json.dumps(dict(parse_qsl(uri_parts.query, keep_blank_values=True)))
 
+    def get_uri(self) -> str:
+        uri = '{}://'.format(self.conn_type.replace('_', '-'))
+
+        user_block = ''
+        if self.login is not None:
+            user_block += quote(self.login, safe='')
+
+        if self.password is not None:
+            user_block += ':' + quote(self.password)
+
+        if user_block > '':
+            user_block += '@'
```

Comment:
```suggestion
            authority_block += '@'
```
In this block you store username and password, so this name will be better and it is compliant with RFC.

Reply:
good point, updated